pyblish_standalone/__main__.py

```diff
@@ -22,6 +22,11 @@ def cli():
                               "can be called multiple times"))
     parser.add_argument("-rh", "--register-host", action="append",
                         help=("Append hosts to register."))
+    parser.add_argument("-rg", "--register-gui", action="append",
+                        help=("Append guis to register."))
+    parser.add_argument("--debug", action="store_true")
+    parser.add_argument("--port", nargs='?',
```

Comment:
Any argument with `--` is optional by default. You can make an argument with `--`, that would make it mandatory, unless you give it an `nargs='?'`.